assets/css/_route_ladder.scss

```diff
@@ -28,24 +43,46 @@
 }
 
 .m-route-ladder__timepoint {
-  align-items: center;
   display: flex;
   justify-content: center;
+  margin-top: -3px;
+
+  &:first-of-type {
+    margin-top: 0;
+    padding-top: 3px;
```

Comment:
Is this needed? Shouldn't the top timepoint be bumped up just as much as the others?

Reply:
The `padding` bit is an attempt to fix a bug on the tablets where the circle is being cutoff. I'm working on seeing this on the tablet now so I can determine if this actually fixes it or not…